styleguide/Tutorials.md

```diff
@@ -0,0 +1,86 @@
+# Tutorial Layout
+
+To create an impactful tutorial guide, you must create a tutorial plan and obtain stakeholder approval for alignment. Adhere to the [Tutorial Plan](#tutorial-plan) section, and the [Tutorial Layout](#tutorial-layout) section to deliver accurate, user-friendly, and impactful tutorials consistently. 
+
+# Table of Contents
+
+ [Tutorial Plan](#tutorial-plan)
+
+ [Tutorial Layout](#tutorial-layout)
+
+ [Markdown Layout](#markdown_layout)
+
+ [Section Breakdown](#section-breakdown)
+
+
+## Tutorial Plan
+
+All tutorials require a tutorial plan document with stakeholder signoff before authoring. Writing the plan out prevents incorrect deliverables and reduces the chance of time spent inefficiently. The tutorial plan process will help you create a much higher-quality tutorial.
+ 
+* To get started, make a copy of the tutorial plan [reference](https://docs.google.com/document/d/1ql_vkaj6dHsobCu0AYuvUXoQ8eojBzetAN3RoGxUepc/edit).
```

Comment:
@rahulhazra97 let's use the tutorial plan template file and add it as a child page and link to it. Update the section respectively. 

https://docs.google.com/document/d/1ql_vkaj6dHsobCu0AYuvUXoQ8eojBzetAN3RoGxUepc/edit#